auth/client/iam/openid4vp.go

```diff
@@ -339,15 +343,19 @@ func (c *OpenID4VPClient) walletWithExtraCredentials(ctx context.Context, subjec
 	}), nil
 }
 
-func (c *OpenID4VPClient) dpop(ctx context.Context, requester did.DID, request http.Request) (string, error) {
+func (c *OpenID4VPClient) dpop(ctx context.Context, requester did.DID, request http.Request) (string, *string, error) {
```

Comment:
weird signature. Unclear what the outputs are and why one is a pointer if it is always used. change to struct?